app/assets/javascripts/application.js.erb

```diff
@@ -176,8 +177,9 @@ function display_node_rows( d ) {
 function display_enhanced_job_data( d ) {
 
     if (is_defined(d.error)) {
-        return      '<div><tr><td>'+ d.error +'</td></tr></div>';
+        return      '<div><tr><td><div class="alert alert-warning">No job details because job has already left the queue. </div></td></tr></div>';
```

Comment:
I might be overthinking it, but I'm returning "INVALID" via the API in case someone attempts to request a random job_id via a GET request. I don't think it's safe to return "C" in that case, but then, if someone is using the API in that manner then they assume their own risk.